examples/C++/DDS/ContentFilteredTopicExample/MyCustomFilter.hpp

```diff
@@ -4,10 +4,17 @@
 #include <fastcdr/Cdr.h>
 #include <fastdds/dds/topic/IContentFilter.hpp>
 
+//! Custom filter class
```

Comment:
Maybe a one sentence description of what is this Custom Filter filtering criteria